### ISSUE TITLE
Bug: package version "next" identified as invalid version

### ISSUE THREAD

**Issue by totkeks** [vscode-contrib/vscode-versionlens#112]
I just tried out the angular 7 beta and therefore installed some packages with the @next suffix. Part of my package.json looks like this then:
```
"dependencies": {
    "@angular/animations": "next",
    "@angular/common": "next",
    "@angular/compiler": "next",
    "@angular/core": "next",
    "@angular/forms": "next",
    "@angular/http": "next",
    "@angular/platform-browser": "next",
    "@angular/platform-browser-dynamic": "next",
    "@angular/router": "next",
    "core-js": "^2.5.4",
    "rxjs": "~6.3.2",
    "zone.js": "~0.8.26"
  },
```

And with version lens active I get this
![image](https://user-images.githubusercontent.com/2941389/45930208-4d077c00-bf5d-11e8-8059-bf15bc6dce3b.png)

I guess the extension should just recognize "next" as a valid version identifier.

**Comment by pflannery** [comment]
@totkeks good spot. it should work for all dist tags

**Comment by pflannery** [comment]
@totkeks Resolved in v0.21.2

**Comment by yeskunall** [comment]
Hi @pflannery, thanks for looking into this so quickly. The fix now says "Fixed to v`x.x.x`" if using `next` versions of packages. What does that exactly mean? Maybe we can look into a better message?

**Comment by pflannery** [comment]
@yeskunall What package are you looking at?
What this usually means is the dist tag used (in your case "next") is not the latest prerelease and therefore acts as if you entered a fixed version. i.e. "1.0.0".

Contrived example:

tag|version|output
-|-|-
rc|2.0.0-rc| "Prerelease 2.0.0-rc"
latest|1.10.0| "Latest"
next|1.0.0-next| "Fixed to 1.0.0-next"


Does that make sense?

**Comment by yeskunall** [comment]
> @yeskunall What package are you looking at?

`gatsby@next`. I understand what you mean. No need to change the message, in that case. Once again, thanks for your work!
